README.md

```diff
@@ -260,9 +260,13 @@ onEditChange(initialData, handleEditChange);
 
 ### Level 4: Enable Managing Blocks directly on your frontend ([TODO](https://github.com/collective/volto-hydra/issues/4))
```

Comment:
Since the main part of the this stage is done you can remove this TODO. extra functions to be added later will have their own TODO links below